tests/flow/test_multi_label.py

```diff
@@ -208,3 +208,18 @@ def test08_multi_label_variable_length_traversal(self):
         query_result = graph.query(query)
         self.env.assertEquals(query_result.result_set, expected_result)
 
+    def test09_test_query_graph_populate_nodes_labels(self):
+        query = """CREATE (a:L1 {v:0})-[:R1]->()"""
```

Comment:
Is this create query really necessary ?

Reply:
yes so we have one node and we want to replace the Label
also it checks that if we found part of the pattern it still merge